src/assets/images/vector/templates-dark.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="#F5F5F5" width="24" height="24">
+    <g><rect fill="none" height="24" width="24"/></g><g><g><g><path d="M2.5,4v3h5v12h3V7h5V4H2.5z M21.5,9h-9v3h3v7h3v-7h3V9z"/></g></g></g>
+</svg>
```

Comment:
[nit] Add a new line

Reply:
Can you kindly tell me that why in other existing svg's, we didn't add new line at the end and why do we need here ?